Containerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian
+
+LABEL org.opencontainers.image.source https://github.com/hulks/meta-nao
+
+RUN apt-get update && apt-get install --no-install-recommends --yes \
+  ca-certificates \
+  curl \
+  file \
+  python3 \
+  xz-utils \
+  zstd \
+  && rm --recursive --force /var/lib/apt/lists/*
+
+ARG version
+
+RUN curl --fail --location --progress-bar --output /opt/toolchain.sh \
+  https://github.com/HULKs/meta-nao/releases/download/${version}/HULKs-OS-$(uname -m)-toolchain-${version}.sh
+
+RUN chmod +x /opt/toolchain.sh && /opt/toolchain.sh -d /naosdk -y && rm /opt/toolchain.sh
+RUN echo . /naosdk/environment-setup-corei7-64-aldebaran-linux >> /root/.bashrc
```

Comment:
https://docs.docker.com/build/building/multi-stage/

Use multi stage builds to only have the SDK in the image. We do not need anything else. The SDK includes all necessary tools for cross-compiling.

Reply:
Thanks for the hint, I now implemented a two-stage build with busybox as a base image  since we need a few basic utils (scratch was not sufficient).

Reply:
Just out of curiosity: What do we need? Why is scratch not sufficient?!

Reply:
Sourcing the environment requires e.g. sed (in /naosdk/sysroots/x86_64-naosdk-linux/environment-setup.d/cmake.sh).